openpype/pipeline/colorspace.py

```diff
@@ -105,7 +156,7 @@ def get_imageio_colorspace_from_filepath(
 
     # match file rule from path
     colorspace_name = None
-    for _frule_name, file_rule in file_rules.items():
+    for _, file_rule in file_rules.items():
```

Comment:
```suggestion
    for file_rule in file_rules.values():
```

Reply:
I do agree with the suggestion, keep the name with underscode (even if is unused), or use `.values()`.